config/locales/client.en.yml

```
@@ -2965,6 +2965,8 @@ en:
           one: "Progress: <strong>%{count}</strong> topic"
           other: "Progress: <strong>%{count}</strong> topics"
         silent: "Perform this action silently."
+        performing: "Performing…"
```

```suggestion
        performing: "Performing bulk operations, please wait…"
```

minor, I just think this is a tiny bit more descriptive